rosetta-source/src/main/rosetta/product-qualification-func.rosetta

```diff
@@ -1211,6 +1216,19 @@ func Qualify_InterestRate_Option_DebtOption: <"Qualifies a product as a Option t
         // qualifies the underlyer of the option as a debt security
 		and optionPayout -> underlier -> security -> securityType all = SecurityTypeEnum -> Debt
 
+func Qualify_InterestRate_ForwardBond: <"Qualifies a product as Interest Rate Bond Forward.">
+    [qualification Product]
+    inputs:
+        economicTerms EconomicTerms (1..1)
+    output:
+        is_product boolean (1..1)
+    alias forwardPayout: economicTerms -> payout -> forwardPayout
+    set is_product:
+        Qualify_AssetClass_InterestRate(economicTerms) = True
+            and forwardPayout -> underlier -> security only exists
+            and (forwardPayout -> settlementTerms -> settlementType any = SettlementTypeEnum -> Physical
+                or forwardPayout -> settlementTerms -> settlementType any = SettlementTypeEnum -> Cash)
```

Comment:
Where does this set of criteria come from ? What happens if the settlement type are election of cashorPhysical ?

Reply:
Given this will impact reporting this logic should be validate by the relevance DRR governance at least.

Reply:
> Where does this set of criteria come from?

@rvincentISDA the logic / criteria is based on the product definition.

> What happens if the settlement type are election of cashorPhysical ?

Good point, will review

Reply:
> Given this will impact reporting this logic should be validate by the relevance DRR governance at least.

Although the driver for this change is related to DRR, this PR is just adding qualification functions for bond forwards according to the ISDA taxonomy.  Once this PR is released, the intention is to add support for bond forwards to DRR, and I agree that that change should be carefully reviewed by DRR governance

Reply:
> What happens if the settlement type are election of cashorPhysical ?

The logic has been updated to only qualify as a bond forward if it's physically settled.  For the other settlement types (Cash, Election, or CashOrPhysical) it does not qualify as a bond forward.

Reply:
@rvincentISDA as discussed, the logic has been updated so it no longer check settlement type, so it still qualifies as a bond forward whether it is cash or physical.